DOC-3039-41 Patch Release 4.7.2 (#503)
corda/corda-docs: content/en/docs/corda-enterprise/4.7/release-notes-enterprise.md

# Corda Enterprise release notes

## Corda Enterprise 4.7.2

Corda Enterprise 4.7.2 is a patch release of Corda Enterprise that introduces fixes to known issues in Corda Enterprise 4.7.1.

### Upgrade recommendation

As a developer, you should upgrade to the [latest released version of Corda](https://docs.corda.net/docs/corda-enterprise/index.html) as soon as possible. Check the latest Corda Enterprise release notes and upgrade guide [here](https://docs.corda.net/docs/corda-enterprise/release-notes-enterprise.html).

As a node operator, you should upgrade to the [latest released version of Corda](https://docs.corda.net/docs/corda-enterprise/index.html) if the fixed issues listed below are relevant to your work.

### Fixed issues

* Corda dependency vulnerability CVE-2020-28052 has been fixed.

> * We have resolved the issue of unbound memory consumption for batched transactions.

Reviewer: ```suggestion
* A fix has been introduced to reduce memory consumption during batched transaction resolution of large backchains.
```

> * We have addressed the Corda 4.6 dependency vulnerability on the component org.bouncycastle:bcprov-jdk15on:1.66. The newer version of this dependency is available which fixes this vulnerability and users should upgrade to address this.

Reviewer 2: I think this should be styled as code. Plus, this could be tightened up a bit. 
```suggestion
* We have addressed the Corda 4.6 dependency vulnerability on the component `org.bouncycastle:bcprov-jdk15on:1.66`. Upgrade to the newer version of this dependency to fix this vulnerability. 
```

Reviewer: We are not advising customers to upgrade. We have made the upgrade - in this patch. So this should be: 

Upgraded to the newer version of this dependency.

> * We have addressed the Corda 4.6 dependency vulnerability on the component org.bouncycastle:bcprov-jdk15on:1.66. The newer version of this dependency is available which fixes this vulnerability and users should upgrade to address this.

Reviewer 3: ```suggestion
* We have addressed the Corda 4.6 dependency vulnerability on the component `org.bouncycastle:bcprov-jdk15on:1.66`. The newer version of this dependency is available which fixes this vulnerability and users should upgrade to address this.
```

> * We have resolved the issue of unbound memory consumption for batched transaction.

Reviewer 2: ```suggestion
* We have resolved the issue of unbound memory consumption for batched transactions.
```
